pages/api.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: API
+permalink: /api/
+---
+
+The Pair Columbus API is a public, read-only resource for information related to the group and its resources. For now, it just serves static JSON used to generate the [Challenges page](/challenges/).
+
+## [Challenges](/api/challenges.json)
+This endpoint returns all active Pair Columbus challenges in categories (generally based on difficultly or technologies used).
+
+### Request
+`GET http://paircolumbus.org/api/challenges.json`
```

Comment:
Using curl, sometimes I get a 200 and sometimes I see this 302:
```
~ → curl --include http://paircolumbus.org/api/challenges.json
HTTP/1.1 302 Found
Connection: close
Pragma: no-cache
cache-control: no-cache
Location: /api/challenges.json

```

I also noticed that caching is disabled by default in Postman. Could this be a cached response?